build.gradle.kts

```diff
@@ -1,7 +1,8 @@
 import com.bmuschko.gradle.docker.tasks.image.DockerBuildImage
 import com.github.gradle.node.npm.task.NpmTask
 import io.smallrye.openapi.api.OpenApiConfig
-import java.util.Locale
+import java.util.*
```

Comment:
We don't want wildcard imports. ;-)

Reply:
We should probably change our IDE settings. That's something that gets me every time I do Kotlin changes

Reply:
Yeah it's a pain. I was hoping we could do it using Spotless maybe?

Reply:
fixed this